daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -285,6 +428,51 @@ func (d *Driver) Create(id, parent, mountLabel string, storageOpt map[string]str
 	return label.Relabel(path.Join(subvolumes, id), mountLabel, false)
 }
 
+// Parse btrfs storage options
+func (d *Driver) parseStorageOpt(storageOpt map[string]string, driver *Driver) error {
+	// Read size to change the subvolume disk quota per container
+	for key, val := range storageOpt {
+		key := strings.ToLower(key)
+		switch key {
+		case "size":
```

Comment:
@calavera Yes. This option is set per container. It can be used like this:
docker create --storage-opt size=10G busybox /bin/sh